LuaUI/Widgets/gui_chili_proconsole2.lua

```diff
@@ -10,7 +10,7 @@ function widget:GetInfo()
 		license      = "GNU GPL, v2 or later",
 		layer        = 50,
 		experimental = false,
-		enabled      = true,
+		enabled      = false,

```

Comment:
What if someone has it already enabled??
May we just remove this widget?

Reply:
If you support the idea, we can (i like where there is just one thing). I kept it there for people who may want to stick with old UI and maybe using some its features (i was not scanning it much).

Widgets loading is based on the config I mentioned in comment - same rules are applied here, player can customize on his side, so he will be using this console instead of the new default one.

We can remove it in the future, not blocking the merge I think.

Reply:
For me you can remove that... You are replacing it with something better, right?

Reply:
I changed my mind... The current console has 2 killer features:

1.- Chat log... You know? a scrollbar to check old messages
2.- When someone add a mark, an icon is added to the console message. Such icon can be used to target the camera to the mark

Probably you can easily import both features to your console

Reply:
* @sanguinariojoe old messages can be checked even in new console vie SHIFT+ESC
* marks are shown in minimap - I understand it may be useful for some people but e.g. I never used it, it was not clear this feature was present in the old one. Also spectators can use the stalking widget to see all actions of some players. Is it really worth of delaying the big thing by this small thing? Keeping the old solution for some time until the replacement is done in the new one can be compromise for few weeks if there are people who would really miss that feature. For majority of others it is not important I believe.

Reply:
We can wait, but in such a case, don't remove the old console... It'll act as a reminder :-p

Reply:
Ok, keep it